### ISSUE TITLE
Serverless output format

### ISSUE THREAD

**Issue by jvargh** [iann0036/former2#94]
Is it possible to get the Serverless output format for selected lambda, api gateway resources?

**Comment by iann0036** [comment]
Hey @jvargh,

Thanks for raising! I'm going to assume you mean Serverless Application Model (SAM), let me know if you mean the other one 😆.

This is actually a pretty complex challenge, as SAM resources are actually comprised of a number of smaller resources so we'd need to:

* ensure that all resources are scanned
* have a way to exclude multiple lower-level resources when in this mode
* determine the correct behaviour when only part of the required resources are present

The same issue is present for CDK constructs also.

tl;dr I have no immediate plans for this and it would require major rework, but I'll keep the request around to gauge support.
